CHANGELOG.md

```diff
@@ -3,13 +3,16 @@ Change Log
 
 This project honours [Semantic Versioning](http://semver.org/).
 
-[Unpublished]: ../../compare/v1.0.0...HEAD
+[Unpublished]: ../../compare/v1.0.1...HEAD
 
 
-[Unpublished]
-------------------------------------------------------------------------
-* **Fixed:** Breakage when loading non-ESM version in browsers
-* **Fixed:** Values not clamped or typecast if set during initialisation
+[v1.0.1]
+ ------------------------------------------------------------------------
+**September 5th, 2022**
+ * **Fixed:** Breakage when loading non-ESM version in browsers
+ * **Fixed:** Values not clamped or typecast if set during initialisation
+
+[v1.0.1]: https://github.com/Alhadis/PanAndZoom/releases/tag/v1.0.1
```

Comment:
Oops, I accidentally counted too many spaces when writing that diff before… 😓 Also, the date and version number is only added right before the release is about to be published. Otherwise, there's a risk that a delay or new problem will come up, and you'll overlook the wrong date and version in the config (I've made that mistake quite a few times 😉).

```suggestion
[v1.0.1]
------------------------------------------------------------------------
**September 5th, 2022**
* **Fixed:** Breakage when loading non-ESM version in browsers
* **Fixed:** Values not clamped or typecast if set during initialisation

[v1.0.1]: https://github.com/Alhadis/PanAndZoom/releases/tag/v1.0.1
```